templates/block/block--inline-block--card-standard.html.twig

```diff
@@ -29,14 +29,14 @@
   {% set label = 'Title not displayed!' %}
 {% endif %}
 <div{{ attributes.addClass('card') }}>
+  {{ title_prefix }}
```

Comment:
Same Q as above: do we need the prefix/suffix without the actual title?

Reply:
Oh, hang on... just seen the label in the template so that's okay. Is there something routinely in the prefix and suffix that we need to combine here in the template, or should this be removed?

Reply:
Yep - I think contextual edit links get rendered by either title_prefix or title_suffix - I think it might be the suffix and there might be a way to specifically output the contextual link.  Will take a look.

Reply:
Yep so - {{ title_suffix.contextual_links }} will output contextual links.  Where we aren't outputting the title, we'll need this token/placeholder in place if the component needs edited in layout builder.